Ground_RealSense_Height: 1.05
Ground_Neck_Height: 0.95
Ground_Arm_Height: 0.78 # 0.48
Shoulder_Elbow_Length: 0.22
Elbow_Wrist_Length: 0.17
Wrist_Endeffector_Length: 0.17

#Gen2  
Origin_Angle: [840, 3260, 2065, 2065, 2150, 1794]
Origin_Pose: [0,0,0]
Carry_Pose: [-85, 90, 90]
Recieve_Pose: [-40, 70, -30]
Give_Pose: [-35, 75, -35]
Point_Pose: [-85, 90, 0]
Crane_Up_Pose: [0, -45, -30]
Crane_Down_Pose: [-33, 0, -40]
Look_Floor_Pose: [-85, 0, 40]
TU_Grasp_Pose: [-85, 90, 5]
TU_Craneup_Pose: []
TU_Cranedown_Pose: []


#Gen3
#Origin_Angle: [830, 3336, 2065, 2065, 2150, 1794]
Gear_Ratio: [2.1, 2.1, 1, 1, 1, 1]
